### ISSUE TITLE
[subviews] Item description scrolling is too limited

### ISSUE THREAD

**Issue by lynxlynxlynx** [gemrb/gemrb#784]
In the item description window, you can only scroll it if you're hovering the text area. IIRC on master we have SetDefaultScrollBar, which got partly replaced by SetFocus. I think this doesn't cut it here, since also the scrollbars get removed from windows if they're attached to textareas (and then autorecreated). But the id is lost, so I don't how to interact with that control (still just hypothetising that SetFocus would help).

TLDR: scrolling over the item image should also scroll the description.
